styles/_utilities.scss

```diff
@@ -0,0 +1,55 @@
+.underline-link {
+    font-weight: $weight--bold;
+
```

Comment:
Again formatting looks to be a little off, might be worth doing this in another ticket when theres less open MRs with css properties

Reply:
Sounds good, will double check the stylelint setup